pkg/volumereplica/volumereplica.go

```diff
@@ -385,6 +389,35 @@ func buildVolumeCloneCommand(cStorVolumeReplica *cstor.CStorVolumeReplica, snapN
 	return cloneVolCmd
 }
 
+func SetTargetIp(fullVolName, targetIp string) error {
+	out, err := zcmd.NewVolumeSetProperty().
+		WithProperty("io.openebs:targetip", targetIp).
+		WithDataset(fullVolName).
+		Execute()
+	if err != nil {
+		klog.Errorf("Unable to set target ip to %s. error : %v %s", fullVolName, err, string(out))
+		return errors.Wrapf(err, "failed to set target ip to %s", fullVolName)
+	}
+
+	return nil
+}
+
+func GetTargetIp(fullVolName string) (string, error) {
+	ret, err := zcmd.NewVolumeGetProperty().
+		WithScriptedMode(true).
+		WithParsableMode(true).
+		WithField("value").
+		WithProperty("io.openebs:targetip").
+		WithDataset(fullVolName).
+		Execute()
+
+	if err != nil {
+		klog.Errorf("Unable to get target ip from %s. error : %v", fullVolName, err)
+		return "", errors.Wrapf(err, "failed to get target ip: %s", fullVolName)
```

Comment:
Any particular reason for logging the error when it is returned? I mean if the error is returned from here the caller will/should handle it.

Reply:
I see, I can remove it. I was just following other methods doing this

Reply:
removed